### DISCUSSION TITLE
2-2. 즉각적인 피드백을 통해 무언가를 쉽게 배울 수 있었던 사례 한 가지 공유하기

### DISCUSSION THREAD

**Discussion by imakerjun** [ssac-dev/yozm#7] [2장 똑똑하게 학습하고 싶어]
# 2-2. 어디서 배워야 할까?

## 1. 지금까지 무언가를 배울 때 가장 효과적이었던 학습 환경 적어보기
무언가를 배울 때 가장 빠르고 효과적으로 학습했던 환경을 공유해주세요.

## 2. 효과적이었던 학습 경험을 만든 조건 세 가지 적어보기
효과적인 학습 경험을 만들어준 조건 3가지를 공우해주세요.

## 3. 세 가지 조건을 만족하는 나만의 학습 환경 찾아보기
위의 3가지 조건을 만족하는 학습 환경에 대해 상세히 설명해주세요.

## 4. 나에게 가장 잘 맞는 학습 환경에서 실천할 수 있는 학습 루틴 적어보기
나에게 가장 잘 맞는 환경에서 실행할 수 있는 나에게 가장 잘 맞는 루틴을 적어주세요.

**Comment by heeji289** [comment]
## 1. 지금까지 무언가를 배울 때 가장 효과적이었던 학습 환경 적어보기

직접 만들어보기, 부딪혀보기가 제일 효과적이었다. 첫 회사에 입사하고나서 제품을 개발하고 유지보수하면서 1달 동안 부트캠프 6개월보다 더 빠른 속도로 배웠다. 또, 이번에 Next.js를 익혀보기 위해 블로그 만들기를 했고 직접 만들기 전에 공식문서 읽기부터 해봤는데 일단 만들면서 필요한 내용을 문서에서 찾는 방법이 제일 효과적이었다.

## 2. 효과적이었던 학습 경험을 만든 조건 세 가지 적어보기

1) 직접 서비스를 만들 수 있다.
2) 물어볼 사람이 있다.
3) 양질의 참고자료가 있다. (공식문서, 고수들의 블로그, 책 등)

## 3. 세 가지 조건을 만족하는 나만의 학습 환경 찾아보기

직접 무언가를 만들면서 마주한 어려움이나 고민을 이야기하고 의견을 들을 수 있는 환경 (회사, 스터디, 슬랙 커뮤니티 등). 또, 적절하게 책이나 잘 작성된 글들을 참고해야 한다. 이 2가지를 적절히 섞는 게 어렵긴 한데 중요해보인다.

## 4. 나에게 가장 잘 맞는 학습 환경에서 실천할 수 있는 학습 루틴 적어보기

사실 회사에서 겪으며 배우는 게 제일 효과적일 것 같은데 지금은 회사를 안다니니까. 만들어보고 싶은 것을 생각해보고 내가 배우고 싶은 기술이나 개념을 적용해서 직접 만들어본다. 이 때, 대충 적용만 하지말고 기술의 컨셉이나 어떻게 하면 잘 쓸 수 있을지 생각해보면 좋겠다.

**Comment by CHPark00** [comment]
**1. 지금까지 무언가를 배울 때 가장 효과적이었던 학습 환경 적어보기**
예를 들어서 학교 공부를 했었을 때. 이론적인 지식을 100% 공부하는 것보다 이론 50% 실전(실습) 50% 이런 방식이 나에게 더 잘되었던 것 같다.

**2. 효과적이었던 학습 경험을 만든 조건 세 가지 적어보기**
-실전(실습)을 무조건 하기
-계속해서. 그리고 꾸준히 할것
-흥미를 붙일 것

**3. 세 가지 조건을 만족하는 나만의 학습 환경 찾아보기**
서로 잘 맞는 친구(동기)들과 함께 공부를 하고 학습환경을 조성하기

**4. 나에게 가장 잘 맞는 학습 환경에서 실천할 수 있는 학습 루틴 적어보기**
개발자 커뮤니티에 가입하기...?
